app/common_handlers.py

```diff
@@ -386,12 +370,8 @@ async def get(self, request):
                 page_title = View.page_title_error_prefix_en + page_title
             locale = 'en'
 
-        attributes = await self.common_check_attributes(request, user_journey, sub_user_journey)
-
-        try:
-            postcode = attributes['postcode']
-        except KeyError:
-            raise SessionTimeout(user_journey, sub_user_journey)
+        attributes = get_session_value(session, 'attributes', user_journey, sub_user_journey)
```

Comment:
Yes, I just went for a generic function which can in fact return any key from the returned session dictionary, or any dictionary object passed to it where you want to raise a SessionTimeout or session error of some sort. Adds a bit of flexibility, for instance to be able to extract the stored case dictionary from the session as well for instance.

Could add a method to call this generic method if you think it really worthwhile?

Reply:
I was suggesting and extra method for the attributes rather than replacing get_session_value .  Anyway it was more a possible suggestion rather than a request to change.